src/store/channels-list/saga.ts

```diff
@@ -110,8 +104,10 @@ export function* fetchConversations() {
     chatClient,
     chatClient.getConversations,
   ]);
+  const otherMembersOfConversations = conversations.map((c) => c.otherMembers);
+
   yield call(mapToZeroUsers, conversations);
-  yield call(updateUserPresence, conversations);
+  yield call(updateUserPresence, otherMembersOfConversations);
```

Comment:
Use `fork` so this doesn't delay the rest of the conversation loading.

Reply:
updated, thanks!